src/ui/json/JsonArray.js

```diff
@@ -18,21 +18,22 @@ class JsonArray extends Component<{
 
   render() {
     const currentIndentation = (
-      <React.Fragment>
-        {Array.from(new Array(this.props.depth).keys()).map((_) => {
-          return <React.Fragment>&nbsp; </React.Fragment>;
-        })}
-      </React.Fragment>
+      <pre
```

Comment:
Looking at this hack... was it considered just use existing open source json editor? The eng cost seems significant.

Reply:
Let me see what kind of open source options we have. I also consider spinning this off as an open source one.

Reply:
One way to build a JSON editor is based on an existing complex editor. This is one example of doing this: http://jsoneditoronline.org/doc/index.html. The downside is pulling in a large open source project. I think I prefer to make a lightweight JSON editor.